src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/OverridePluginFunctionalSpec.groovy

```diff
@@ -1,80 +1,92 @@
 package info.solidsoft.gradle.pitest.functional
 
+import groovy.transform.CompileDynamic
 import nebula.test.functional.ExecutionResult
 import org.gradle.api.GradleException
 import spock.lang.Issue
 import spock.lang.PendingFeature
 
+@CompileDynamic
```

Comment:
Most of the Spock specification cannot be used togheter with `@CompileStatic`. It might be expressed explicitly to make Codenarc happy.

Reply:
That's right - would you like to disable the rule then and only use `@CompileStatic` where possible (and not in Spock specifications)

Reply:
It's fine with that